processors/config/rawSvtAnaHits_cfg.py

```diff
@@ -0,0 +1,71 @@
+import HpstrConf
+import sys,os
+import baseConfig as base
+
+options = base.parser.parse_args()
+
+# Use the input file to set the output file name
+root1_file = options.inFilename[0]
+root2_file = options.outFilename
+
+print('Root file Input: %s' % root1_file)
+print('Root file Output: %s' % root2_file)
+
+p = HpstrConf.Process()
+
+p.run_mode = 1
+p.skip_events = 0
+p.max_events = 200000
```

Comment:
It would be better to add these as options to the arg parser instead of hard coding them like this.

Reply:
Did it